### DISCUSSION TITLE
Change reference caption when using format book.

### DISCUSSION THREAD

**Discussion by OffByOne37** [quarto-dev/quarto-cli#9198] [Q&A]
### Description

Hello,

i want to change the bibliography caption in my book output. 
I know there exists "biblio-title" but this does not work since i use a csl. 
In the pandoc documentation i saw "reference-section-title" but as far as i know this is the same command as "biblio-title" for quarto. 

Does someone know how I can change this? (Latex commands for \renewcommand\bibname or \refname do not work either)

Thanks!

**Comment by cderv** [comment]
If I understand correctly, you are referring to the header above the bibliography list right ? I am not sure if I got `reference caption` right. 

If this is it, did you try changing the part above the bibliography when including it (https://quarto.org/docs/authoring/footnotes-and-citations.html#bibliography-generation) 

````markdown
### Custom section

::: {#refs}
:::
````

![image](https://github.com/quarto-dev/quarto-cli/assets/6791940/c544e961-16e5-4f5a-9930-5e9f5c168685)


This works in HTML document when using Pandoc's citeproc and CSL. It should in books too. 
From: https://quarto.org/docs/books/book-structure.html#references
> Note that you can change the chapter title to whatever you like, remove .unnumbered to have it be numbered like other chapters, and add other content before or after the bibliography as necessary.

**Comment by mcanouil** [comment]
You can use `language` and set any translation you like as documented in <https://quarto.org/docs/authoring/language.html>.

Or use the section/filename as described in https://quarto.org/docs/books/book-structure.html#references

<table>
<tr><th>Quarto document</th><th>HTML</th></tr>
<tr><td>

````qmd
---
title: "Quarto Playground"
format: html
language:
  section-title-references: "My Amazing References"
references:
- type: article-journal
  id: WatsonCrick1953
  author:
  - family: Watson
    given: J. D.
  - family: Crick
    given: F. H. C.
  issued:
    date-parts:
    - - 1953
      - 4
      - 25
  title: 'Molecular structure of nucleic acids: a structure for
    deoxyribose nucleic acid'
  title-short: Molecular structure of nucleic acids
  container-title: Nature
  volume: 171
  issue: 4356
  page: 737-738
  DOI: 10.1038/171737a0
  URL: https://www.nature.com/articles/171737a0
  language: en-GB
---


@WatsonCrick1953
````

</td><td>

<img width="494" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/e272cc5c-ac14-4c96-898d-69e201a7f333">


</td></tr>
</table>


If this does not work for you, please provide a small reproducible example to work with.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.
